src/main/java/michael/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package michael;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    private LocalDate deadline;
+
+    public Deadline(String taskName, String deadline) {
+        super(taskName);
+        this.deadline = LocalDate.parse(deadline);
+    }
+
+    public LocalDate getDeadline() {
+        return this.deadline;
+    }
+
+    @Override
+    public String toString() {
+        String formatted_deadline = this.deadline.format(DateTimeFormatter.ofPattern("d MMM yyyy"));
```

Comment:
Could the variable name be changed to follow camelCase standards?
```suggestion
        String formattedDeadline = this.deadline.format(DateTimeFormatter.ofPattern("d MMM yyyy"));
```